frontend/TestRun/ResourcesInfo.svelte

```diff
@@ -1,7 +1,14 @@
 <script>
     export let resources;
+    export let backend;
+    export let test_id;
```

Comment:
I would instead keep it consistent and call it `runId` here, and just pass that to the `test_id` jenkins parameter.

Reply:
fixed